src/components/TrackCreateModal/TrackCreateModal.tsx

```diff
@@ -1,71 +1,62 @@
 import React from "react";
-import { useSelector } from "react-redux";
-import { RootState } from "../../store";
-import TrackForm from "../TrackForm/TrackForm";
-import { createTrack } from "../../features/tracks/trackSlice";
-import { toast } from "react-toastify";
 import { useAppDispatch } from "../../hooks/redux-hook";
+import { createTrack } from "../../features/tracks/trackSlice";
+import { selectAllTracks } from "../../features/tracks/trackSlice";
```

Comment:
You have two imports from "../../features/tracks/trackSlice" which can me made 1.